src/lib/Layout/PanelSideBarLayout/PanelSideBar/PanelSideBarItem.tsx

```diff
@@ -23,6 +23,13 @@ const PanelSideBarItem = (props: PanelSideBarItemProps) => {
   if (item.display === false) {
     return null;
   }
+  const scollToActiveItemRef = useRef<HTMLDivElement>(null);
+
+  useEffect(() => {
+    if (scollToActiveItemRef.current && (item.children?.find((s) => s.active) || item.active)) {
+      scollToActiveItemRef.current.scrollIntoView();
+    }
+  }, [scollToActiveItemRef]);
```

Comment:
@severinoAmmirati Seems strange this dependency on the ref, it should not be allowed by eslint rule. Are you sure this is needed?

Reply:
@manni497 Not required. Removed